Formula/yacas.rb

```diff
@@ -17,8 +17,7 @@ class Yacas < Formula
   def install
     args = ["--disable-silent-rules",
             "--disable-dependency-tracking",
-            "--prefix=#{prefix}"
-           ]
+            "--prefix=#{prefix}"]
```

Comment:
Would you mind making this a slightly neater style of array?
```ruby
%W[
  --disable-silent-rules
  --disable-dependency-tracking
  --prefix=#{prefix}
]
```
We prefer that style when possible because, well, it looks a lot cleaner.